keps/sig-node/4540-strict-cpu-reservation/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: Add CPUManager policy option to restrict reservedSystemCPUs to system daemons and interrupt processing
+kep-number: 4540
+authors:
+  - "@jingczhang"
+owning-sig: sig-node
+participating-sigs:
+  - sig-scheduling
+status: implementable
+creation-date: 2024-03-06
+reviewers:
+  - "@klueska"                                                                          
+  - "@swatisehgal"                                                                      
+approvers:
+  - "@sig-node-tech-leads"                                                              
+  - "@soltysh"
```

Comment:
I'm not the KEP approver, just PRR, so remove me from here.

Reply:
sure